src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,99 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__header {
+		display: flex;
+	}
+
+	.tribe-events-calendar-month__header-column {
+		text-transform: uppercase;
+		width: 14.285%;
+		padding-bottom: var(--spacer-2);
+	}
+
+	.tribe-events-calendar-month__week {
+		display: flex;
+
+		&:first-of-type {
+			@media (--viewport-medium) {
+				border-top: 1px solid var(--color-border-default);
+			}
+		}
+		@media (--viewport-medium) {
+			border-left: 1px solid var(--color-border-default);
+		}
+	}
+
+	.tribe-events-calendar-month__day {
+
+		width: 14.285%;
+
+		@media (--viewport-medium) {
+			border-width: 0 1px 1px 0;
+			border-style: solid;
+			border-color: var(--color-border-default);
+			min-height: 168px;
+			@mixin desktop-heading-4;
+		}
+
+		&:hover,
+		&:focus {
+			@media (--viewport-medium) {
+				border-bottom: 1px solid var(--color-border-active);
+				box-shadow: inset 0 -1px 0 0 var(--color-border-active);
+			}
+		}
+	}
+
+	.tribe-events-calendar-month__day--current {}
+
+	.tribe-events-calendar-month__day--disabled {}
+
+	.tribe-events-calendar-month__day-date {
+		@mixin heading;
+		@mixin heading-6;
+
+		@media (--viewport-medium) {
+			@mixin desktop-heading-4;
+		}
+
+		padding: var(--spacer-1);
+
+		a {
```

Comment:
The problem here is that the link will be shown only for days where there are events. If I follow that then we'll have to make logic on the template, or set the same class for both elements (with the reset, links don't have the cursor: pointer).

Reply:
Even if we only show links for those days that have events, we should still target them with classes. We are already doing `if` and `for` logic on templates, and there is always going to be this sort of logic.

Let's add the `cursor: pointer` to the reset then so we don't have to repeat this.

Reply:
I still don't see the point of this. Having the same styles for different classes, duplicating just for the sake of having a particular class for a hyperlink that is inside many elements. ¯\_(ツ)_/¯

Reply:
It's more about consistency, future maintainability and also provides a nice set of css "hooks" for our users as well and makes their customizations cleaner : ) I'd rec having the same class on the <a> for the shared styles regardless of state and then add a modifier class to either the day with or without the event.

Reply:
If I put in the customer shoes and I have to change two classes with the same values to set the month day number, I won't understand why. Or if I'm a client I changed the way the "current" day looked by inspecting the element and then when the current day has events my styles don't take effect, because there are two classes.

Reply:
Should be like that or a frustrating experience depending on how we set this up. Let's chat about this 😉